.devcontainer/rocm-azp-agent-in-docker/rocm-azp-agent.dockerfile

```diff
@@ -0,0 +1,78 @@
+# Use the AMD ROCm image as the base image
+FROM rocm/dev-ubuntu-24.04@sha256:708bbb9e2031a81e949a0847f0655c4fb1a790f8ac7cd0031116f56b04ada7d5
```

Comment:
You can set this to the ROCm devcontainer. Then it will take less time to build.